### ISSUE TITLE
[BUG]: Missing first level heading on home page

### ISSUE THREAD

**Issue by colabottles** [TechIsHiring/techishiring-website#146]
## Expected Behavior

The page in question does not have a `<h1>` element as the indicator of where the user is on the page and as a landmark for screen reader users and users that use the keyboard for navigation.

## Current Behavior

The lack of a level heading one may result in confusion or disorientation when a user is navigating the page in question.

## Possible Solution

Hierarchy of heading levels for screen reader and users of keyboard for navigation.

## Context (Environment)

It's in production and it fails my audit for the page.

## Detailed Description

Adding a heading level one to the page for screen reader users and keyboard navigating users to easily parse the page with screen reader or keyboard.

## Possible Implementation

Use the `<h1>` element to surround the _"TechIsHiring"_ text in the header.

**Comment by chadstewart** [comment]
Hey @yiremorlans, sure feel free to take on this issue!
